### PR TITLE
[HOPSWORKS-3300] Change the location of pem files

### PR DESCRIPTION
**Pull request by gibchikafa**
If we will use hopsfs-mount for Jupyter, the pem files should be written to a temporary directory in the container not in HopsFs

### CODE REVIEW

File: hops/constants.py [hunk 1]
```diff
@@ -88,6 +88,7 @@ class SSL_CONFIG:
     K_CERTIFICATE_CONFIG = "k_certificate"
     T_CERTIFICATE_CONFIG = "t_certificate"
     PEM_CLIENT_CERTIFICATE_CONFIG = "client.pem"
+    PEM_DIR = "/tmp"
```
Comment (kouzant) [inline hops/constants.py:91]: Do not assume that the temporary directory will always be `/tmp` Use Python's build-in function https://docs.python.org/3/library/tempfile.html#tempfile.gettempdir

File: hops/util.py [hunk 2]
```diff
@@ -535,6 +537,37 @@ def write_b64_cert_to_bytes(b64_string, path):
...
+    if kernel_id != "":
+        logging.info("Kernel id for {} is {}".format(relative_path, kernel_id))
+        try:
+            attach_jupyter_configuration_to_notebook(kernel_id)
+            logging.info("Notebook configuration successfully attached for file {} attached".format(relative_path))
+        except Exception as e:
+            logging.error("Failed to attach notebook configuration: " + str(e))
+    else:
+        logging.error("No kernel id found for notebook " + relative_path)
+    return
```
Comment (kouzant) [inline hops/util.py:553]: Return statement here is not needed

File: hops/constants.py (hunk 1 above)
Comment (kouzant) [reply hops/constants.py:91]: Also, why do you need to change the location? I don't know if there are other libraries that assume the extracted pem files are in the current working directory. Can you please verify with Moritz and Till?

**Review by kouzant — CHANGES_REQUESTED** [review]

File: hops/constants.py (hunk 1 above)
Comment (kouzant) [reply hops/constants.py]: @gibchikafa have you confirmed with @moritzmeister and @tdoehmen that it does not affect hsfs or flyingduck?

Comment (tdoehmen) [reply hops/constants.py]: We use `hsfs.client._get_ca_chain_path()` etc in the flyingduck client. As long as that returns the correct path it should be fine

**Pull request merged**